app/views/logs/delete_logs.html.erb

```diff
@@ -0,0 +1,58 @@
+<% title = "Delete logs" %>
+<% content_for :title, title %>
+<% content_for :before_content do %>
+  <%= govuk_back_link(href: :back) %>
+<% end %>
+
+<h1 class="govuk-heading-l">
+  <span class="govuk-caption-l"><%= title %></span>
+  Review the logs you want to delete
+</h1>
+<p>You've selected <%= @delete_logs_form.log_count %> <%= "log".pluralize(@delete_logs_form.log_count) %> to delete</p>
+
+<%= form_with model: @delete_logs_form, url: @delete_logs_form.delete_confirmation_path do |f| %>
+  <%= f.hidden_field :search_term, value: @delete_logs_form.search_term %>
+  <%= f.govuk_error_summary %>
+  <%= govuk_table do |table| %>
+    <% table.head do |head| %>
+      <% head.row do |row| %>
+        <% row.cell(header: true, text: "Log ID") %>
+        <% if @delete_logs_form.log_type == :lettings %>
```

Comment:
There is enough variation here that you should render a different template for each table, embedded within the form.

Reply:
I just merged `delete_lettings_logs.html.erb` and `delete_sales_logs.html.erb` yesterday as after I had polished everything then they had two lines different.
I think I will need a little convincing to change this back as as far as I can tell what you're suggesting is still having an if/else inside the form to render two different table partials that only have 2 lines out of ~30 different, so it feels like much of a muchness to me.

Reply:
You could render the template based on the type, which would avoid the need for a conditional statement, for example:

```ruby
<%= render template: "delete_#{@delete_logs.form.log_type}" %>

Reply:
aha ok cool. will do.
out of interest, I would expect that to be `render partial: ...`
a quick scan of docs and SO makes me think it doesn't massively matter the difference?

Reply:
Sorry, yes, typo!